modules/dnn/src/tensorflow/tf_importer.cpp

```diff
@@ -747,3 +747,22 @@ Ptr<Importer> cv::dnn::createTensorflowImporter(const String&)
 }
 
 #endif //HAVE_PROTOBUF
+
+Net cv::dnn::readNetFromTensorflow(const String &model)
+{
+    Ptr<Importer> tensorflowImporter;
+    try
+    {
+        tensorflowImporter = createTensorflowImporter(model);
+    }
+    catch (cv::Exception& e)
+    {
+        std::cerr << "Exception: " << e.what() << std::endl;
+        return Net();
+    }
```

Comment:
I don't like this design.
How user application can handle a error and detect "not implemented" or "I/O error" error types?
Why do not pass exception directly to user code?